app/models/event.py

```diff
@@ -262,6 +265,14 @@ def __setattr__(self, name, value):
         else:
             super(Event, self).__setattr__(name, value)
 
+    def set_default_event_image(self, event_topic_id):
+        if self.event_topic_id is None:
```

Comment:
What is the point of having a parameter when you are not using it? or using it some place. Better to remove the parameter from method and the property of the class.

Reply:
How about 
```
def set_default_event_image(self, event_topic_id):
        if event_topic_id is None:
            return None
        else:
            event_topic = EventTopic.query.filter_by(
                event_topic_id=event_topic_id).first()
            return event_topic.system_image_url
```

Reply:
I think, since we are validating an attribute before creating the object in __init__ method Constructor so cannot use `self.event_topic_id` in creation.